doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -441,6 +441,27 @@ Here are examples:
    end
    where "n + m" := (plus n m).
 
+Deactivating notations
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. cmd:: Notation Set {? {| @string | @qualid } } {? := @one_term } {? ( {+, @notation_set_flag } ) } {? {| : no scope | : @ident } }
+         Notation Unset {? {| @string | @qualid } } {? := @one_term } {? ( {+, @notation_set_flag } ) } {? {| : no scope | : @ident } }
```

Comment:
> Since the right hand sides are the same, how about combining them

Right, done

> I see everything on the right hand side is optional. Is that syntactically correct? Are there semantic checks that something that some of them are present? (What, if anything would Notation Set. do?

Added a sentence and documented the error is nothing is present.

> Also, would you rename @ident to @scope, @scope_name or @scope_key to correspond to its semantics?

I'd love to but I was unable to find the correct `REPLACE` statement to add in `common_edit.mlg`